src/utilities/util_list.c

```diff
@@ -135,15 +135,14 @@ list_print(WT_SESSION *session, const char *uri, bool cflag, bool vflag)
 
         /*
          * !!!
-         * We don't normally say anything about the WiredTiger metadata
-         * since it's not an application/user "object" in the database.
-         * I'm making an exception for the checkpoint and verbose
-         * options. However, skip over the metadata system information
+         * We don't normally say anything about the WiredTiger metadata and history store since
+         * they are not an application/user "objects" in the database. I'm making an exception for
```

Comment:
You didn't write this comment, but it would be better to change it from first person.

```
Don't report anything about the WiredTiger metadata and history store since they are not user created objects unless the verbose or checkpoint options are passed in. ...
```

Reply:
Fixed using the provided statement.